src/snapred/backend/recipe/algorithm/DiffractionSpectrumWeightCalculator.py

```diff
@@ -36,6 +36,7 @@ def PyExec(self):
                 "Predicting peaks...",
                 InstrumentState=self.getProperty("InstrumentState").value,
                 CrystalInfo=self.getProperty("CrystalInfo").value,
+                PeakIntensityThreshold=0.00,
```

Comment:
You want to set this as a string: 
`PeakIntensityThreshold="0"`

Reply:
Done!